wigwam/_utils.py

```diff
@@ -203,6 +203,24 @@ def test_image(image: Image, expression: str) -> bool:
             raise
 
 
+def get_libdir(
+    base_tag: str,
+) -> str:
```

Comment:
Weird that this got wrapped onto 3 lines.

```suggestion
def get_libdir(base_tag: str) -> str:
```